src/libAtomVM/opcodesswitch.h

```diff
@@ -1990,19 +1990,23 @@ HOT_FUNC int scheduler_entry_point(GlobalContext *glb)
                             // Support compilers < OTP26 that generate CALL_EXT
                             // for min/2 and max/2
                             const struct Bif *bif = EXPORTED_FUNCTION_TO_BIF(func);
```

Comment:
BTW, is this part of code handling non-existing BIFs gracefully?

Reply:
Not really, but it is on my radar as well.